barretenberg/cpp/src/barretenberg/api/api.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+#include <filesystem>
+#include <iostream>
+
+namespace bb {
+
+class API {
+  public:
+    struct Flags {
+        bool verbose{ false };
+        bool debug{ false };
```

Comment:
some comments on all of these might help

Reply:
very nit: I would have just done = false, = "" etc which is equivalent and a bit more readable